lib/connection/result/result.js

```diff
@@ -101,6 +101,7 @@ function Result(options) {
     ColumnNamesCreator.addOverridenNamesForDuplicatedColumns(rowtype);
   }
 
+  Logger.getInstance().trace(`Mapping columns in resultset (total: ${numColumns})`);
```

Comment:
We are only interested in quantity?

Reply:
for now; yes. the request was to somehow log column mapping progress